tests/test_schema.py

```diff
@@ -72,6 +72,26 @@ def testSourceSchemaMatch(self):
         dataId = {"instrument": "LSSTCam-imSim", "detector": 100, "visit": 5884, "band": "y"}
         self._validateSchema("sourceTable_visit", dataId, "source")
 
+    def testForcedSourceSchemaMatch(self):
+        """Check forcedSourceTable_tract"""
+        dataId = {"instrument": "LSSTCam-imSim", "tract": 0, "skymap": "discrete/ci_imsim/4k"}
+        self._validateSchema("forcedSourceTable_tract", dataId, "forcedSource")
```

Comment:
Don't need to change anything, but note for the future that: `forcedSourceTable_tract` and `forcedSourceOnDiaObjectTable_tract` may not exist for very long.